libs/common/src/cast.helper.ts

```
@@ -523,3 +523,20 @@ export function ValidateNestedStructureFields(validationOptions?: ValidationOpti
     });
   };
 }
+
+export function buildUrlWithQuery<T extends Record<string, any>>(baseUrl: string, queryParams: T): string {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)